src/main/scala/com/fulcrumgenomics/util/NumericTypes.scala

```diff
@@ -146,7 +146,7 @@ object NumericTypes {
     }
 
     /**
-      * Precise computation of log(1-exp(x)) for use in the `aOrNotB` method.
+      * Precise computation of log(1-exp(-x)) for use in the `aOrNotB` method.
       * See Equation (7) in https://cran.r-project.org/web/packages/Rmpfr/vignettes/log1mexp-note.pdf
```

Comment:
This is correct, though the R docs says it's for `x > 0`. So I think line `153` should be `if (value <= 0)` with an updated comment